lib/tests/streamlit/multiselect_test.py

```diff
@@ -23,12 +23,7 @@
 from streamlit.errors import StreamlitAPIException
 from tests import testutil
 from enum import Enum
-
-
-class Drinks(Enum):
```

Comment:
It could be a good idea to also include a test using an `Enum` defined in this file, that checks for the `StreamlitAPIException` being raised correctly, since it is important that developers are notified.

As an additional benefit, if Streamlit ever changes something fundamentally, avoiding the memory location issue entirely, this test would fail, signifying that your defensive code could be removed.